_docs/privacy/web-tracking-protections.md

```diff
@@ -66,10 +66,10 @@ To protect against this, we first block most embedded 3rd-party tracking request
 
 | Platform          | Support                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Firefox extension | 3rd-party scripts on our list blocked from creating or accessing 1st-party cookies, including those impersonating 1st parties on our CNAME list. Automatic 24-hour expiry enforced for 1st-party cookies set by scripts on our list. Automatic 7-day expiry enforced for all other 1st-party cookies created by all other scripts.                                                                                                                                                                                                                              |
```

Comment:
> Automatic 7-day expiry enforced for all ~~other~~ 1st-party cookies created by all other scripts.                                                                                      

I think “other” should be removed from each of these sentences, as we’re now removing the sentence that it refers to.